lxc/action.go

```diff
@@ -202,6 +202,10 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 		state = true
 	}
 
+	if action == "stop" && c.flagForce {
```

Comment:
Oh, that's missing a `&& c.flagConsole` :)

Reply:
Oh, that's my bad. I will fix in the morning if that's ok.

Reply:
Yep, no worries.

Reply:
@stgraber Should I re-run `make i18n` after I have changed the if statement?

Reply:
No, this doesn't change the translated string so no need to re-run.

Reply:
Hi @stgraber, I think I have made a mistake and the commit to fix the if statement is in this commit: `i18n: Update translation templates. ` Will this be a problem?

Reply:
I think I have kind of answered my own question, I will try fix and it.

Reply:
All done. Apologies for the mess.